charts/operator/Chart.yaml

```diff
@@ -56,3 +56,7 @@ dependencies:
     repository: https://charts.stakewise.io
     version: "0.1.0"
     condition: lighthouse.enabled
+  # - name: teku
+  #   repository: https://charts.stakewise.io
+  #   version: "1.0.0"
```

Comment:
Shouldn't the version be here similar to others? `0.1.0`

Reply:
ok :)